src/AccessTokenEvents.ts

```diff
@@ -36,17 +37,17 @@ export class AccessTokenEvents {
                     expiring = 1;
                 }
 
-                this._logger.debug("load: registering expiring timer in:", expiring);
+                logger.debug("registering expiration timer");
```

Comment:
why not logging `expiring`?

Reply:
the log needed better phrasing and it felt simpler to just remove the value since it's already logged in the `init()` method immediately following this. I can add it back with some better phrasing